test/test_parser.py

```diff
@@ -113,5 +111,52 @@ def test_txtmeta_parse():
             }
         ]
     }
-    ddiff = DeepDiff(dictmeta, expected)
-    assert not ddiff
+
+    assert not DeepDiff(actual, expected)
+
+
+def test_autonode():
+    """
+    Test the automatic creation of implicit nodes
+    """
+
+    topology = """
+    sw1:port1
+    """
+
+    actual = parse_txtmeta(topology)
+
+    expected = {
+        'environment': OrderedDict(),
+        'nodes': [{'attributes': OrderedDict(), 'nodes': ['sw1']}],
+        'ports': [{'ports': [('sw1', 'port1')], 'attributes': OrderedDict()}],
+        'links': []
+    }
+
+    assert not DeepDiff(actual, expected)
+
+
+def test_multiline():
+    """
+    Test the support for multiline attributes
+    """
+
+    topology = """
+    # Environment
```

Comment:
The spaces before `[` make this (and the other) test case fail. This is inconsistent with the previous behavior of pyszn.

Reply:
Fixed